pkg/apis/core/v1beta1/types.go

```diff
@@ -270,8 +270,19 @@ type CDISpec struct {
 	Infra sdkapi.NodePlacement `json:"infra,omitempty"`
 	// Restrict on which nodes CDI workload pods will be scheduled
 	Workloads sdkapi.NodePlacement `json:"workload,omitempty"`
+	// Clone strategy override: should we should we use a host-assisted copy even if snapshots are available?
+	// +kubebuilder:validation:Enum="copy"
+	CloneStrategyOverride *CDICloneStrategy `json:"cloneStrategyOverride,omitempty"`
 }
 
+// CDICloneStrategy defines the preferred method for performing a CDI clone (override snapshot?)
+type CDICloneStrategy string
+
+const (
+	// CloneStrategyHostAssisted specifies slower, host-assisted copy
+	CloneStrategyHostAssisted = "copy"
```

Comment:
Even though it is the default, I would add the smart clone option as well. I know it will not actually do anything, but having everything listed somewhere will help people understand what we are doing.